sunpy/physics/differential_rotation.py

```diff
@@ -25,12 +25,14 @@
 from sunpy.sun.constants import sidereal_rotation_rate
 from sunpy.time import parse_time
 from sunpy.util import expand_list
+from sunpy.util.decorators import deprecated
 from sunpy.util.exceptions import warn_user
 
 __all__ = ['diff_rot', 'solar_rotate_coordinate', 'differential_rotate']
 
 
 @u.quantity_input
+@deprecated( since="6.0",message="We will be moving this", alternative="sunpy.sun.models.diff_rot")
```

Comment:
```suggestion
@deprecated(since="6.0",message="We will be moving this", alternative="sunpy.sun.models.diff_rot")
```

What does the deprecation warning look like?

Reply:
Not sure. I copied the line from the documentation and changed the values of parameters.

Reply:
Can you share/check what the warning looks like?

Reply:
when I ran the tests this was the output
```FAILED sunpy/physics/tests/test_differential_rotation.py::test_rigid - sunpy.util.exceptions.SunpyDeprecationWarning: We will be moving this```

Reply:
In that case, the message doesn't really offer much.

Reply:
Should I make it more detailed?

Reply:
That message should be removed.